Analysis/Utkscan/core/source/UtkUnpacker.cpp

```diff
@@ -60,6 +60,9 @@ void UtkUnpacker::ProcessRawEvent(ScanInterface *addr_/*=NULL*/) {
         PrintProcessingTimeInformation(systemStartTime, times(&systemTimes),
             GetEventStartTime(), eventCounter);
 
+    //set eventWidth from Globals (loaded from config file)
+    eventWidth = Globals::get()->GetEventLengthInTicks();
```

Comment:
Actually, this should happen even before that. I think it needs to go in [this ](https://github.com/pixie16/paass/blob/master/Scan/utkscan/core/source/UtkScanInterface.cpp#L64) method. Putting it into UtkUnpacker::InitializeDriver() is too late since it's already started to build events.